### PR TITLE
Markus Doan-Le completed string-manipulation-1

### PR DESCRIPTION
**Pull request by mdoanle**
Screen Shot 2022-09-05 at 3 45 58 PM

### CODE REVIEW

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  var isUpper = 0;
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-lower-cased.js:3]: This should just be a boolean

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  var isLower = 0;
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-upper-cased.js:3]: This should just be a boolean

File: string-manipulation-1/src/is-vowel.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/* exported isVowel */
+function isVowel(char) {
+  var upperCaseVowels = ['A', 'E', 'I', 'O', 'U'];
+  var lowerCaseVowels = ['a', 'e', 'i', 'o', 'u'];
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:4]: If you just convert the argument character to either uppercase or lowercase, you can make it so you only need to check one or the other, not both.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/is-vowel.js [hunk 4]
```diff
@@ -1,35 +1,20 @@
...
+    return true;
+  } else if (testingVariable === upperCaseVowels[2]) {
+    return true;
+  } else if (testingVariable === upperCaseVowels[3]) {
+    return true;
+  } else if (testingVariable === upperCaseVowels[4]) {
+    return true;
+  } else {
+    return false;
   }
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:18]: You should convert this to a loop

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/is-vowel.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  for (var i = 0; i < upperCaseVowels.length; i++) {
+    if (testingVariable === upperCaseVowels[i]) {
+      booleanOutput = true;
+      continue;
+    } else if (testingVariable !== upperCaseVowels[i]) {
+      booleanOutput = false;
+    }
+  }
+  return booleanOutput;
+}
```
Comment (codingcodymiller) [inline string-manipulation-1/src/is-vowel.js:15]: Did you test this?
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: string-manipulation-1/src/is-vowel.js (hunk 5 above)
Comment (mdoanle) [reply string-manipulation-1/src/is-vowel.js:15]: Sorry! I was still in progress but didnt want to waste my progress.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**